types/dsinjs__binary-tree/tsconfig.json

```diff
@@ -0,0 +1,31 @@
+{
+  "compilerOptions": {
+    "target": "es5",
+    "module": "commonjs",
+    "lib": [
+      "ES2018",
```

Comment:
Why were all of these needed?

Reply:
I guess basic lib like dom, ES2017 will also work, will update these changes.